processing/data_collection/gazette/spiders/mg_itauna.py

```diff
@@ -0,0 +1,36 @@
+import datetime as dt
+from dateparser import parse
+from scrapy import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgItaunaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3133808"
+    name = "mg_itauna"
+    allowed_domains = ["itauna.mg.gov.br"]
+    start_urls = ["https://www.itauna.mg.gov.br/portal/diario-oficial/"]
+
+    def parse(self, response):
+        url = self.start_urls[0] + "{i}/0/0/0/0/"
+        last_page = int(response.xpath("//select/option/text()")[-1].extract())
+        for i in range(1, 1 + last_page):
+            yield Request(url.format(i=i), callback=self.parse_editions_page)
+
+    def parse_editions_page(self, response):
+        diarios = response.xpath("//div[@class='d_e_modelo_diario']")
```

Comment:
**issue**:
It is a bit dangerous to search for a class using XPath like that. This will try to find a `div` where the class is **exactly** equal to `d_e_modelo_diario`. 

`response.xpath("//div[@class='d_e_modelo_diario']")` will match:

`<div class="d_e_modelo_diario"><!-- My DIV content --></div>`

But it will not match (look the extra whitespace):
`<div class="d_e_modelo_diario "><!-- My DIV content --></div>`

It will not match this one either:
`<div class="d_e_modelo_diario another_class"><!-- My DIV content --></div>`

So a single whitespace may break everything. For class search, I prefer to use a CSS Selector, that avoid this problem:

`diarios =response.css("div.d_e_modelo_diario")` or `diarios =response.css(".d_e_modelo_diario")` if this is the only tag with this calss.

Reply:
True. That was my mistake.